app/helpers/public_pages_helper.rb

```diff
@@ -0,0 +1,7 @@
+module PublicPagesHelper
+  def enable_online_intake?
+    return true unless Rails.env.production?
+
+    ENV['ENABLE_ONLINE_INTAKE'].present? && session[:source] != 'propel'
```

Comment:
for `session[:source]` we should instead use the getter method, `source` that's available on the ApplicationController, just so that we have a consistent way to get the source.

Reply:
^ that was ben, i missed this

Reply:
there were some complications here, but good idea on the general approach of re-using that method. I had to call it as `controller.source` from within the helper, and also it was a bit tricky in test because the TestController class that Rspec sets up for us doesn't inherit from ApplicationController. fyi @bengolder  see https://github.com/codeforamerica/vita-min/pull/80/commits/b2f3896e3fc68ad02a9c5d8cc4ea31bbea4c7225 for the diff if you're curious.